portfolio/src/main/webapp/script.js

```diff
@@ -43,27 +41,35 @@ function createImg(imageName) {
   return imgElement;
 }
 
-/**
- * Fetches the new comment and builds the UI.
- */
+/** Sets num of comments to be displayed when a page loads. */
+function load() {
+  const urlParams = new URLSearchParams(window.location.search);
+  const maxComments = urlParams.get('maxComments');
+  if (maxComments == '5' || maxComments == '10') {
```

Comment:
Why only in these cases?

Reply:
see my answer to your comment on DataSevlet.. it applies here also

Reply:
Ack.